### DISCUSSION TITLE
Welcome to ww-assets-postman-collection Discussions!

### DISCUSSION THREAD

**Discussion by Gappster** [WoodWing/ww-assets-postman-collection#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maui0815** [comment]
Hello, the video seems to be gone and it was very helpful for the first steps. Any chance to get it back? Thanks!

**Reply by Gappster** [reply]
Hmmmm... strange. Let me see what I can do otherwise I'll create a new one...
